python/cudf/dataframe/dataframe.py

```diff
@@ -160,6 +160,12 @@ def dtypes(self):
         return pd.Series([x.dtype for x in self._cols.values()],
                          index=self._cols.keys())
 
+    @property
+    def shape(self):
+        """Returns a tuple representing the dimensionality of the DataFrame.
+        """
+        return len(self._index), len(self._cols)
```

Comment:
Rather than checking the `len(self._index)` we should check `len(self)`

Reply:
👍